src/commands/crashes/lib/mdfind.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * Node module for searching OSX Spotlight, using the built-in mdfind shell command.
+ * Forked from brandonhorst/node-mdfind github repo.
```

Comment:
add reason to the copy (insecure deps if I remember correctly)

Reply:
done.